python-api/src/core/llm/schemas.py

```
@@ -0,0 +1,76 @@
+    role: Literal["system", "user", "assistant"]
+    content: str
+
+
+class ChatRequest(BaseModel):
+    """
+    Provider-agnostic chat completion request.
+
+    Attributes:
+        model:           Model identifier string (e.g. ``"openai/gpt-4o"``).
+        messages:        Ordered list of conversation turns.
+        temperature:     Sampling temperature (0–2).
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)